src/__test__/dist/editTask.test.dev.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+var _editTasks = require("../__mocks__/editTasks");
+
+/* eslint-disable no-undef */
```

Comment:
- [ ] Hi, the reason you have to do make this eslint-disable comments is because you have the wrong eslint config file. in your repository you have a config file called `.eslintrc.js` that is not the correct file. it should be a `.eslintrc.json`. please check the linters config repository and use the correct file.